core/src/main/java/com/linecorp/armeria/server/RouteBuilder.java

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright 2019 LINE Corporation
+ *
+ * LINE Corporation licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.linecorp.armeria.server;
+
+import static java.util.Objects.requireNonNull;
+
+import com.linecorp.armeria.common.HttpRequest;
+import com.linecorp.armeria.common.HttpResponse;
+
+/**
+ * A builder class for building a {@link Service} fluently. This class can only be created through
+ * {@link ServerBuilder#route()}.
+ *
+ * <p>Call {@link #service(Service)} to build the {@link Service} and return to the {@link ServerBuilder}.
+ *
+ * <pre>{@code
+ * ServerBuilder sb = new ServerBuilder();
+ * sb.route().get("/foo/bar")                      // Configure the first service.
+ *   .consumes(JSON, PLAIN_TEXT_UTF_8)
```

Comment:
Perhaps we could indent at `.get()` to highlight the level?

Reply:
Fixed, thanks!